src/rhsmlib/dbus/objects/register.py

```diff
@@ -28,6 +31,42 @@
 log = logging.getLogger(__name__)
 
 
+def temporary_disable_dir_watchers(watcher_set: set = None) -> None:
```

Comment:
I wonder whether this method (and `enable_dir_watchers()`) should be added to some more common code: I see two similar methods:
- src/rhsmlib/dbus/objects/config.py:def temporary_disable_dir_watcher():
- src/rhsmlib/services/syspurpose.py:def temporary_disable_dir_watcher():

that basically do the same, just for a single watcher, and IMHO they could be simplified by having this function in a common place, and calling it with their watcher.

Reply:
Done in the last commit